pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
```

Comment:
Do you mean "even if there are no quotas defined", or "even if quotas are defined, but not used" (not sure the latter actually makes sense)?